pages/common/od.md

```diff
@@ -0,0 +1,25 @@
+# od
+
+> Display a file contents in hexadecimal, octal, or printable character format.
+> Optionally display the byte offsets into the file.
+
+- Display file in octal format with the byte offsets in octal as well:
+
+`od --address-radix=o --format=o -v  {{/path/to/file}}`
```

Comment:
What are the defaults for these options? Whenever possible, pages should start with the simplest invocations first and add options gradually, as described in the [contribution guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).